lib/travis/api/v3/services/enterprise_license/find.rb

```diff
@@ -1,19 +1,11 @@
 module Travis::API::V3
   class Services::EnterpriseLicense::Find < Service
     def run!
-      if replicated_endpoint
-        # We turn off verification because this is an internal IP and a self signed cert so it will always fail
-        http_options = {url: replicated_endpoint, ssl: Travis.config.ssl.to_h.merge(verify: false)}.compact
-        conn = Faraday.new(http_options) do |conn|
-          conn.use OpenCensus::Trace::Integrations::FaradayMiddleware if Travis::Api::App::Middleware::OpenCensus.enabled?
-          conn.adapter Faraday.default_adapter
-        end
-        response = conn.get("license/v1/license")
-        replicated_response = JSON.parse(response.body)
-        license_id = replicated_response["license_id"]
-        license_type = replicated_response["license_type"]
-        seats = get_seats(replicated_response)
-        expiration_time = replicated_response["expiration_time"]
+      if replicated_license
+        license_id = Travis.config.replicated.license_id
```

Comment:
I am not sure that it is a config, but let's get a feedback from other developers.